OgreMain/include/OgreRoot.h

```diff
@@ -931,7 +931,7 @@ namespace Ogre
 
         /** Get the default minimum pixel size for object to be rendered by
         */
-        Real getDefaultMinPixelSize() { return mDefaultMinPixelSize; }
+        Real getDefaultMinPixelSize() const { return mDefaultMinPixelSize; }
```

Comment:
please create a separate commit for `const` changes in contrast to API neutral whitspace/ for refactorings

Reply:
done